ResourceStatusSystem/Utilities/CSHelpers.py

```diff
@@ -248,53 +248,25 @@ def getStorageElementEndpoints( storageElements = None ):
 
 def getFTS():
   '''
-    Gets all storage elements from /Resources/FTSEndpoints
+    Gets all FTS endpoints
   '''
 
-  ftsEndpoints = []
-
-  fts2 = getFTS2()
-
-  if not fts2['OK']:
-    return fts2
-
-  ftsEndpoints += fts2['Value']
-
-  fts3 = getFTS3()
-
-  if not fts3['OK']:
-    return fts3
-
-  ftsEndpoints += fts3['Value']
-
-
+  ftsEndpoints = _getFTSEndpoints('Resources/FTSEndpoints/FTS2')
```

Comment:
Why FTS2 ?

Reply:
I don't know. This part has been there since long, I have just re-shuffled it now. But if you think that these locations are wrong, I'll just drop/change them. 

At the same time, talking about locations: what is the "/Resources/FTSEndpoints/Default" for? Which one is used?

Reply:
it was used for FTS2. 
Well okay, we did not officially deprecate completely FTS2 did we ? In that case I propose you leave it there, you add Default on top of that (so we fully support FTS2 still), and as soon as the new agent is certified, good bye for good FTS2, these sections can go

Reply:
OK done.